src/client/apps/edit/components/admin/components/article/article_authors.tsx

```diff
@@ -1,24 +1,18 @@
-import request from "superagent"
+import { Box, Flex } from "@artsy/palette"
+import Input from "@artsy/reaction/dist/Components/Input"
 import { clone, uniq } from "lodash"
+import React, { Component } from "react"
 import { connect } from "react-redux"
+import request from "superagent"
 import { difference, flatten, pluck } from "underscore"
-import PropTypes from "prop-types"
-import React, { Component } from "react"
-import { Col, Row } from "react-styled-flexboxgrid"
-import { onChangeArticle } from "client/actions/edit/articleActions"
-import { AutocompleteList } from "client/components/autocomplete2/list"
-import AutocompleteListMetaphysics from "client/components/autocomplete2/list_metaphysics"
-import { AuthorsQuery } from "client/queries/authors"
-
-export class ArticleAuthors extends Component {
-  static propTypes = {
-    article: PropTypes.object,
-    apiURL: PropTypes.string,
-    isEditorial: PropTypes.bool,
-    onChangeArticleAction: PropTypes.func,
-    user: PropTypes.object,
-  }
+import { onChangeArticle } from "../../../../../../actions/edit/articleActions"
+import { AutocompleteList } from "../../../../../../components/autocomplete2/list"
+import AutocompleteListMetaphysics from "../../../../../../components/autocomplete2/list_metaphysics"
```

Comment:
@eessex - we shouldn't be using anything but absolute paths everywhere; relative paths like this are a nightmare to refactor. Are you running into `tsconfig.json` issues?

Reply:
None of the tsx files are resolving correctly in tests, added a ticket to address here- https://artsyproduct.atlassian.net/browse/GROW-1086

Reply:
Do they resolve correctly in source code?

Reply:
Yeah they are fine in the component files

Reply:
I added a test file and everything seemed good: https://github.com/artsy/positron/pull/1849

Reply:
Lol, well that blew up... on sec